src/commands/helm-chart-template.yaml

```
@@ -41,4 +41,5 @@ steps:
   - run:
       name: "architect/helm-chart-template: Cleanup"
       command: |
-          rm -v .build_*
+          [ ! -f .build_tagged_operator_build ] && echo "skip: not tagged operator build" && exit 0
+          rm -rv .build_*
```

Sorry it should be force:

```suggestion
          rm -fv .build_*
```

See this:

```
bash-5.0$ touch pawel.txt
bash-5.0$ rm -fv pawel*
removed 'pawel.txt'
bash-5.0$ rm -fv pawel*
bash-5.0$ rm -rv pawel*
rm: cannot remove 'pawel*': No such file or directory
bash-5.0$ rm -fv pawel*
bash-5.0$
```

---

Done. Thanks for checking.

Testing in https://github.com/giantswarm/chart-operator/pull/368 to be sure all issues resolved.

---

Works as did the chart linter! 🎉